krummstab/commands/init.py

```diff
@@ -406,8 +562,15 @@ def init(_the_config: config.Config, args) -> None:
     # ├── Team 12345
     # .   └── Muster_Hans_hans.muster@unibas.ch_000000
     # .       └── submission.pdf or submission.zip
-    validate_team_dirs(_the_config, sheet_root_dir)
-    create_all_submission_info_files(_the_config, sheet_root_dir)
+    submission_teams = read_teams_from_adam_spreadsheet(sheet_root_dir)
+    use_names_from_config(_the_config.teams, submission_teams)
+    validate_teams(_the_config, list(submission_teams.values()))
```

Comment:
I think the exercise marking mode doesn't have to be integrated into the checks made here. For the exercise marking mode the teams in the config should not have to matter at all, that is, they shouldn't have to be defined at all. I'm not sure if this is actually the case everywhere else in the code, but we should not force teams to be defined correctly here. The only necessary check I can think of for the exercise marking mode is to verify that the groups don't have more members than is allowed, but this only needs to look at submission teams. I'd suggest writing a different validation function for the exercise marking mode and only cover the static case in the validate procedure that is already implemented.